### DISCUSSION TITLE
CC2531 coordinator LED always off?

### DISCUSSION THREAD

**Discussion by rocket59** [Koenkk/zigbee2mqtt#8242] [General]
Just upgraded z2m to the latest version from a fairly old one (can't recall which one now)

Previously the led on my CC2531 was always on when z2m was running. 

Now it turns off once z2m is started but turns on again when joining is permitted using the front end. Can someone confirm this is the correct behaviour?

**Comment by Koenkk** [comment]
Yes this is the correct behaviour.

**Reply by rocket59** [reply]
Thank you!
